### PR TITLE
chore: move kb articles from tavultesoft.com

### LINKED ISSUE #76: Move kb content from tavultesoft.com to help.keyman.com
- [x] Add a Markdown to HTML preprocessor
- [x] Reformat each KB article as Markdown
- [x] Setup web.config to redirect /kb/xxxx to appropriate /kb/index.php?id=xxxx to load the markdown.
- [x] Setup redirects from tavultesoft.com

### PR DESCRIPTION
**Pull request by mcdurdin**
Closes #76.

### CODE REVIEW

**Comment by mcdurdin** [comment]
Okay, so much of the content is outdated. Correcting the outdated content is not the primary purpose of this PR; the purpose is moving the existing content over in order to retire tavultesoft.com. Thus, please don't reject on basis of irrelevant content!

See also https://github.com/mcdurdin/tavultesoft.com/pull/39

File: knowledge-base/kb0003.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+<p><b>Microsoft Office XP and Microsoft Office 2003</b>
+<br>Supports the supplementary plane characters. Supports Unicode under Windows NT and newer. Also supports Unicode under Windows 95/98/ME with the <a href="https://keyman.com/downloads/">WordLink add-in</a>.</p>
+
+<p><b>Microsoft Office 2000</b>
+<br>Like Office 97, supports Unicode characters in the basic multilingual plane (under U+FFFF) reasonably well. Unicode support is dependent on the operating system. Under Windows NT and newer, Office applications handle Unicode without trouble. Otherwise, supports Unicode input with Keyman and the <a href="https://keyman.com/downloads/">WordLink add-in</a> under Windows 95, Windows 98, and Windows ME.</p>
+
+<p><b>Microsoft Office 97</b>
+<br>Supports Unicode characters in the basic multilingual plane (under U+FFFF) reasonably well. Unicode support is dependent on the operating system. Under Windows NT and newer, Office applications handle Unicode without trouble. Otherwise, Word 97 supports Unicode input with Keyman and the <a href="https://keyman.com/downloads/">WordLink add-in</a> under Windows 95, Windows 98, and Windows ME.</p>
```
Comment (darcywong00) [inline knowledge-base/kb0003.md]: Does the **WordLink add-in** executable need to be transferred to the downloads page? There's 3 links on this page

**Comment by darcywong00** [comment]
Can `knowledge-base/assets/kb0061/Thumbs.db` be removed?

**Comment by darcywong00** [comment]
Re: `developer/whitepaper1.1.pdf`

The links to the white paper at the bottom of developer/9.0 ... developer/12.0 and developer/9.0/guides/distribute/tutorial/step-6.php can be updated this.

File: knowledge-base/kb0040.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
+# HOWTO: Send a support diagnostic to Tavultesoft
+
+### **NOTE**: This archived documentation has not been updated recently and may contain information that is no longer relevant
+
+
+<p>Tavultesoft Support may ask you to send a support diagnostic. These reports assist us in resolving your support issues. Please follow the steps below to generate a diagnostic report.</p>
+
+<div style='border:1px maroon solid; background-color:#f3e5de; padding:0px 5px;'>
+<p><b>Note:</b> If Keyman is not currently installed on your computer, you can download the diagnostic tool as a standalone here: <a href='/support/tsysinfo.exe'>tsysinfo.exe</a> (1845KB) [<a href='/support/tsysinfo.zip'>zipped version</a> (816KB)]</p></div>
```
Comment (darcywong00) [inline knowledge-base/kb0040.md]: Do these links need to be updated/removed?

File: knowledge-base/kb0045.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+
+<h2>Symptoms</h2>
+
+<p>When using a non-Unicode system shadow keyboard (that is, a Keyman keyboard installed through Windows' Regional and Language Options) on Windows Vista, the characters are transformed through an unexpected codepage which causes the output to be corrupted.</p>
+
+<h2>Resolution</h2>
+
+<p>A utility is available from Tavultesoft Support to resolve this problem.  Please contact Tavultesoft Support (<a href='mailto:support@tavultesoft.com'>support@tavultesoft.com</a>) to obtain this utility and instructions on how to run it.</p>
+
```
Comment (darcywong00) [inline knowledge-base/kb0045.md]: I don't see what references `knowledge-base/assets/kb0045/kb0045.zip` or `.exe`.

File: knowledge-base/kb0003.md (hunk 1 above)
Comment (mcdurdin) [reply knowledge-base/kb0003.md]: It's so old we no longer support it. I could move it over but it doesn't work with any recent versions of Word and is only needed on Win 95/98/Me anyway. Not gonna worry about it :)

File: knowledge-base/kb0040.md (hunk 2 above)
Comment (mcdurdin) [reply knowledge-base/kb0040.md]: Now pointing to downloads.keyman.com (see keymanapp/downloads.keyman.com#25)

Comment (mcdurdin) [reply knowledge-base/kb0040.md]: (this is one article it would be good to update because we still use it)

File: knowledge-base/kb0045.md (hunk 3 above)
Comment (mcdurdin) [reply knowledge-base/kb0045.md]: It was a private download; we won't worry about it for now.

**Pull request merged**